mcs/class/Mono.Cairo/Mono.Cairo/SurfaceType.cs

```diff
@@ -44,5 +45,18 @@ public enum SurfaceType
 		BeOS,
 		DirectFB,
 		Svg,
+		OS2,
+		Win32Printing,
+		QuartzImage,
+		Script,
+		Qt,
+		Recording,
+		VG,
+		GL,
+		Drm,
+		Tee,
+		Xml,
+		Skia,
+		SubSurface
```

Comment:
Looks like you're mirroring an enum which I don't see in my copy of cairo. Could you point it to us as a comment?